taxi_service/settings.py

```diff
@@ -39,6 +39,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
```

Comment:
The `debug_toolbar` should only be included in `INSTALLED_APPS` when `DEBUG` is `True`. Consider making this conditional to avoid exposing sensitive information in production.